python/paddle/fluid/contrib/slim/tests/qat2_int8_image_classification_comparison.py

```diff
@@ -0,0 +1,348 @@
+#   copyright (c) 2019 paddlepaddle authors. all rights reserved.
```

Comment:
2020

Reply:
@sfraczek Since windows ci is not stable now, how about merge this PR at first, and update the above typo in the next PR?